strings/base_coroutine_threadpool.h

```diff
@@ -85,21 +85,26 @@ namespace winrt::impl
         check_hresult(context->ContextCallback(resume_apartment_callback, &args, guid_of<ICallbackWithNoReentrancyToApplicationSTA>(), 5, nullptr));
     }
 
+    struct threadpool_resume
+    {
+        threadpool_resume(com_ptr<IContextCallback> const& context, coroutine_handle<> handle) :
+            m_context(context), m_handle(handle) { }
+        com_ptr<IContextCallback> m_context;
+        coroutine_handle<> m_handle;
+    };
+
+    inline void __stdcall fallback_submit_threadpool_callback(void*,
+                                                            void* p) noexcept
```

Comment:
```suggestion
    inline void __stdcall fallback_submit_threadpool_callback(void*, void* p) noexcept
```

Consistent formatting.